expression/builtin_op.go

```diff
@@ -37,8 +37,10 @@ var (
 )
 
 var (
-	_ builtinFunc = &builtinLogicAndSig{}
-	_ builtinFunc = &builtinLogicOrSig{}
+	_ builtinFunc = &builtinIntLogicAndSig{}
```

Comment:
Do we have to remove `ast.And` and `ast.Or` from `canFuncBePushed` since the implementation of these two functions are not consistent with `TiKV` now?
What's your opinion @XuHuaiyu ？

Reply:
I refer to this pr, https://github.com/pingcap/tidb/pull/10498 . ShouId keep the `builtinLogicAndSig` signature compatible to tiKV?